packages/perseus/src/widgets/__tests__/explanation.test.ts

```diff
@@ -133,9 +133,9 @@ describe("Explanation", function () {
         renderQuestion(question1);
 
         // Act
-        const expandLink = screen.getByRole("button", {expanded: false});
+        const expandLink = screen.getByTestId("expand-button");
         userEvent.click(expandLink); // expand and then
-        const collapseLink = screen.getByRole("button", {expanded: true});
+        const collapseLink = screen.getByTestId("expand-button");
```

Comment:
Is there a way we could avoid using a test ID here, e.g. by selecting the element by text instead? My understanding is that if a feature is hard to test without a test ID, it's likely inaccessible (users can't select elements by test ID, after all). I don't think there's an accessibility problem here—just want to promote exemplary testing practices.

Reply:
Good catch. I re-implemented the use of `getByRole()` with an updated content reference.